### ISSUE TITLE
Can't remove friends (unless you know what stands there)

### ISSUE THREAD

**Issue by greenbigfrog** [crmarsh/discord-linux-bugs#79]
![image](https://cloud.githubusercontent.com/assets/6208894/17658392/bce7c214-62cb-11e6-8c38-a85f4c4f01c2.png)
The `remove` is getting cut off on the `Friends` List when I have it sized for half off one of my 2 monitors...

As size reference:
![image](https://cloud.githubusercontent.com/assets/6208894/17658433/022b0656-62cc-11e6-8d43-f626e68095dd.png)

**Comment by greenbigfrog** [comment]
Does this also occur on other OSes and builds or is it only me on canary?

**Comment by crmarsh** [comment]
Yeah, that happens everywhere. Basically, there is a minimum width to the window, but we found that enforcing it strictly introduced worse bugs than it fixed. Ideally we would fix up the design to allow for scaling properly to narrower windows, but you know, designer time is finite.

**Comment by greenbigfrog** [comment]
Yep... Expected a similar answer

**Comment by greenbigfrog** [comment]
![image](https://cloud.githubusercontent.com/assets/6208894/17806934/c0783398-6607-11e6-8f06-cb3e233c2c86.png)
Similar Issue caused by the width (Gonna change topic)

**Comment by tkdberger** [comment]
So if it's on all platforms, maybe we should move the issue elsewhere? I don't know if Discord in general has an issue tracker like this, but if it does then this likely doesn't belong here (correct me if I'm wrong though, please.)

**Comment by crmarsh** [comment]
It is cross platform; we don't have any other sort of public bug tracker, though. I was leaving this one open here in the hope that other people who might otherwise open more bugs about it would see this.
